okapi-core/src/main/java/org/folio/okapi/util/DepResolution.java

```diff
@@ -736,7 +661,58 @@ static void installMaxIterations(
     if (!errors.isEmpty()) {
       throw new OkapiError(ErrorType.USER, String.join(". ", errors));
     }
-    sortTenantModules(tml, modsAvailable, modsEnabled);
+    sortTenantModules(tml, modsAvailable, enabledModules);
+
+  }
+
+  static void sortTenantModules(List<TenantModuleDescriptor> tml,
```

Comment:
Might be good to have a javadoc here. I'm left wondering what the sort is for. The error message below says its a "topological sort" -- seems important what that is. If it is topological what's the topic, etc.

Reply:
Agreed. Added some javadoc to the method. This is fun stuff :-)